tests/sentry/demo/test_demo_org_manager.py

```diff
@@ -25,7 +25,7 @@
 
 
 class DemoOrgManagerTeest(TestCase):
-    @override_settings(DEMO_ORG_OWNER_EMAIL=org_owner_email)
+    @override_settings(DEMO_MODE=True, DEMO_ORG_OWNER_EMAIL=org_owner_email)
```

Comment:
You can set `@override_settings` on a class as well. Does it make sense to make a `DemoModeTestCase` where we set this, then you don't need to set it on every test you write?

Could do the same thing for any views, have `DemoModeViewTestCase` and set the urlconf, although not sure how many views you're likely to have.

Reply:
@wedamija good call! Just made that change